python/cudf/cudf/core/series.py

```diff
@@ -1490,12 +1491,21 @@ def __neg__(self):
     @copy_docstring(CategoricalAccessor.__init__)
     @property
     def cat(self):
-        return self._column.cat(parent=self)
+        if not is_categorical_dtype(self.dtype):
+            raise AttributeError(
+                "Can only use .cat accessor " "with a 'category' dtype"
+            )
```

Comment:
Should this check be pushed down into the `CategoricalAccessor` init instead?

Reply:
Done